src/main/scala/xiangshan/mem/lsqueue/LoadQueue.scala

```diff
@@ -210,7 +211,8 @@ class LoadQueue(implicit p: Parameters) extends XSModule
   val loadQueueReplay = Module(new LoadQueueReplay)  //  enqueue if need replay
   val virtualLoadQueue = Module(new VirtualLoadQueue)  //  control state
   val exceptionBuffer = Module(new LqExceptionBuffer) // exception buffer
-  val uncacheBuffer = Module(new UncacheBuffer) // uncache buffer
+  val ioBuffer = Module(new IOBuffer) // uncache io buffer
+  val ncBuffer = Module(new NCBuffer) // uncache nc buffer
```

Comment:
Is it necessary to use two large buffers to save MMIO access and uncache access separately?